db/migrate/20220705215942_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
+      t.integer :comments_counter
+      t.string :likes_counter
+      t.string :integer
```

Comment:
- [ ] Kindly remove the `integer` column since we did not have it, Take a look at the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) given 🤓

Reply:
Thanks updated